addons/api/addon/handlers/indexed-db-handler.js

```diff
@@ -102,6 +102,9 @@ export default class IndexedDbHandler {
           if (payload.removed_ids?.length > 0) {
             await indexedDb[type].bulkDelete(payload.removed_ids);
           }
+          if (!storeToken) {
```

Comment:
Maybe leave a quick comment about why this hack was needed

Reply:
thx, good point!